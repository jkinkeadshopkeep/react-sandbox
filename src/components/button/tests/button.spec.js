import React from 'react';
import { render } from '@testing-library/react';
import Button from '..';

describe('Button Component', () => {
  it('should render button', () => {
    const { container } = render(<Button>My Button</Button>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
