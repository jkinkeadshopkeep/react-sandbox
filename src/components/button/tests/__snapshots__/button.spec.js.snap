// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component should render button 1`] = `
<button
  type="button"
>
  My Button
</button>
`;
